rs/system_api/src/routing.rs

```diff
@@ -332,30 +328,30 @@ fn route_idkg_message(
 
     match requested_subnet {
         Some(subnet_id) => match network_topology.subnets.get(subnet_id) {
-            None => Err(ResolveDestinationError::IDkgKeyError(format!(
+            None => Err(ResolveDestinationError::ChainKeyError(format!(
                 "Requested threshold key {} from unknown subnet {}",
                 key_id, subnet_id
             ))),
             Some(subnet_topology) => {
                 if subnet_topology.idkg_keys_held.contains(key_id) {
```

Comment:
Eventually we should also rename this (`SubnetTopology::idkg_keys_held`) to, e.g., `SubnetTopology::chainkeys_held`, but that can be done in another PR.